Vizzuality/marxan-cloud
# more API scaffolding - CORS setup

### Overview

Add setup for CORS.

### Designs

N/A

### Testing instructions

Spin up a local http daemon, serving the content of the `api/test/html` folder, e.g.

```
cd api/test/html && busybox httpd -p 8989 -f
```

Add `http://localhost:8989` as allowed origin via `.env`:

```
[...]
NETWORK_CORS_ORIGINS=http://localhost:8989
```

Visit http://localhost:8989/cors-test.html: the content of the API request in the HTML file (`/api/v1/users`) should be displayed on the page (and there should not be any CORS errors related to this API call shown in the devtools console).

Also, HTTP headers set by Helmet should be visible in the response headers in devtools.

### Feature relevant tickets

https://www.pivotaltracker.com/story/show/176205745

---

## Checklist before submitting

      deploying to staging/production, please add brief testing instructions
      to the deploy checklist (`docs/deployment-checklist.md`)
